supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -29,12 +29,16 @@ In the *Name* field, enter `test-postgresql`.
 [[user-interface-elements]]
 == User interface elements
 
-Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names, dialog box names, and window names. Bold the item, even if it is not clickable.
+Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
```

Comment:
```suggestion
Use bold text for all graphical user interface (GUI) element names, including menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
```
Redundant and gets rid of an extra layer of commas.

Reply:
Updated. Also made the items plural to keep it simpler and read a little better (I think at least). Let me know what you think!